hera_cal/lstbin.py

```diff
@@ -417,10 +416,11 @@ def lst_bin_arg_parser():
     a.add_argument("--silence", default=False, action='store_true', help='stop feedback to stdout')
     a.add_argument("--output_file_select", default=None, nargs='*', help="list of output file integers ot run on. Default is all output files.")
     a.add_argument("--vis_units", default='Jy', type=str, help="visibility units of output files.")
+    a.add_argument("--ignore_flags", default=False, action='store_true', help="Ignore flags in input data files.")
```

Comment:
Can you be a little more verbose here? What does "ignore flags" mean? All data is treated as unflagged and the flags on the lst-binned product are all False?

Reply:
it means all input data are included in the binning. i'll add that to the description